website/blueprints/hds-component-docs/files/tests/acceptance/components/__name__.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'website/tests/helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+
+module('Acceptance | components/<%= dasherizedModuleName %>', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /components/<%= dasherizedModuleName %>', async function (assert) {
+    await visit('/components/<%= dasherizedModuleName %>');
+
+    assert.strictEqual(currentURL(), '/components/<%= dasherizedModuleName %>');
+  });
+
+  test('Components/<%= dasherizedModuleName %> page passes a11y automated checks', async function (assert) {
```

Comment:
```suggestion
  test('Components/<%= dasherizedModuleName %> page passes automated a11y checks', async function (assert) {
```

Reply:
I think we have an inconsistency between how these are worded in the components project vs the website project. Happy to make this change, but future website tests will not match the previous website tests.